### ISSUE TITLE
CiviSEPA 1.6.0 might crash with payment processors

### ISSUE THREAD

**Issue by bjendres** [Project60/org.project60.sepa#630]
It looks like there is an issue with the consolidation of the SEPA entity names. In combination with ``org.project60.sepapp`` the following error can be observed (with a :
```
Error: Class name must be a valid object or a string in _civicrm_api3_basic_get() (line 1261 of [...]/httpdocs/pro/public/sites/all/modules/civicrm/api/v3/utils.php).
```
The origin of is an early ``SepaCreditor.get`` call than cannot find the corresponding BAO of the ``SepaCreditor`` entity.

Since this has worked well in all other test scenarios, I'm assuming at this point that the call happens *before* the ``civicrm_entityTypes`` hook, which defines the correct name.

**Comment by bjendres** [comment]
I have provided a quick fix for this, but it should be investigated if the early call in ``org.project60.sepapp`` should be delayed.

**Comment by bjendres** [comment]
released with ``1.6.1``
